mozilla/foundation.mozilla.org
# RSS Reader Fix

Closes #7261 

**Link to sample test page**: https://foundation-s-7261-rss-e-iaa1ux.herokuapp.com/en/blog/rss/

**Steps to test**:

1. Visit the link above 
2. Add it to your rss reader of choice
3. The page should render correctly with no 500 error and should appear in the RSS reader correctly as well.
**Note:** Due to an ongoing redirect bug on review apps, it appears that all links are redirecting to a mozfest URL instead of the review apps correct URL. 
Due to this, my RSS reader seemed to be rendering the pages correctly:
<img width="338" alt="Screen Shot 2021-09-01 at 2 23 10 PM" src="https://user-images.githubusercontent.com/18314510/131747201-3741572c-0671-4cac-99b4-c9ce86467959.png">

But will link to a bad URL if clicked on.

## Checklist

**Changes in Models:**
